tox.ini

```diff
@@ -21,7 +21,7 @@ deps =
     pylint
 commands =
     pylint grammarinator
-    pycodestyle grammarinator --ignore=E501 --exclude=grammarinator/parser/ANTLRv4*.py
+    pycodestyle grammarinator --ignore=E501 --exclude=grammarinator/tool/g4/ANTLRv4*.py
```

Comment:
The `omit` field in `[coverage:run]` section should also be updated to avoid coverage decrease.

```ini
[coverage:run]
omit = **/tool/g4/*
```

Reply:
It works indeed. Fixed.